src/aws/osml/model_runner/database/job_table.py

```diff
@@ -20,41 +19,52 @@
-    width: Optional[Decimal] = width of the image
-    height: Optional[Decimal] = height of the image
-    feature_distillation_options: Optional[str] = the options used in selecting features (NMS/SOFT_NMS, thresholds, etc.)
-    roi_wkt: a Well Known Text representation of the requested processing bounds
+    image_id: str = unique identifier for the image associated with the job
+    job_id: Optional[str] = unique identifier for the job
+    image_url: Optional[str] = S3 URL or another source location for the image
+    image_read_role: Optional[str] = IAM role ARN for accessing the image from its source
+    model_invoke_mode: Optional[str] = mode in which the model is invoked (e.g., batch or streaming)
+    start_time: Optional[int] = time in epoch milliseconds when the job started
+    expire_time: Optional[int] = time in epoch seconds when the job will expire
+    end_time: Optional[int] = time in epoch milliseconds when the job ended
+    region_success: Optional[int] = current count of regions that have successfully processed for this image
+    region_error: Optional[int] = current count of regions that have errored during processing
+    region_count: Optional[int] = total count of regions expected for this image
+    width: Optional[int] = width of the image in pixels
+    height: Optional[int] = height of the image in pixels
+    extents: Optional[str] = string representation of the image extents
+    tile_size: Optional[str] = size of the tiles used during processing
+    tile_overlap: Optional[str] = overlap between tiles during processing
+    model_name: Optional[str] = name of the model used for processing
+    outputs: Optional[str] = details about the job output
+    processing_duration: Optional[int] = time in seconds taken to complete processing
+    feature_properties: Optional[str] = additional feature properties or metadata from the image processing
+    feature_distillation_option: Optional[str] = the options used in selecting features (e.g., NMS/SOFT_NMS, thresholds)
+    roi_wkt: Optional[str] = a Well-Known Text (WKT) representation of the requested processing bounds
     """
 
     image_id: str
     job_id: Optional[str] = None
     image_url: Optional[str] = None
     image_read_role: Optional[str] = None
     model_invoke_mode: Optional[str] = None
-    start_time: Optional[Decimal] = None
-    expire_time: Optional[Decimal] = None
-    end_time: Optional[Decimal] = None
-    region_success: Optional[Decimal] = None
-    region_error: Optional[Decimal] = None
-    region_count: Optional[Decimal] = None
-    width: Optional[Decimal] = None
-    height: Optional[Decimal] = None
+    start_time: Optional[int] = None
+    expire_time: Optional[int] = None
+    end_time: Optional[int] = None
+    region_success: Optional[int] = None
+    region_error: Optional[int] = None
+    region_count: Optional[int] = None
+    width: Optional[int] = None
+    height: Optional[int] = None
```

Comment:
Did DynamoDB made an update recently to accept the `int` property?

Reply:
We can discuss but it always accepted int properties - it just defaulted to converting numerical values coming out of DDB into Decimals - which is why some of the code was convoluted with that type.